docs/reference/limitations/openstack.md

```diff
@@ -20,6 +20,15 @@ It also means that you cannot use the following `openstack` CLI commands; they a
   > You can use `openstack project list`, but this will only list the project(s) that your user account has access to.
 * `openstack domain <subcommand>`
 
+## Nova
+
+### Nested virtualization
+
+Running nested virtualization is only supported for Nova instances running Linux as the guest operating system.
```

Comment:
Nova instances act like hosts, right? If so, then I would suggest this change:
```suggestion
Nested virtualization only works in Nova instances running Linux.
```

Reply:
Yes, can do.